ppml/trusted-deep-learning/base/Dockerfile

```diff
@@ -26,6 +47,13 @@ ADD ./pert_ipex.py   /ppml/examples/pert_ipex.py
 ADD ./filelock.patch /filelock.patch
 ADD ./bash.manifest.template.patch /bash.manifest.template.patch
 
+# COPY frontend.jar
+COPY --from=temp /ppml/torchserve/frontend.jar /ppml/torchserve/frontend.jar
+# Examples for Torchserve
+ADD ./start-backend-sgx.sh      /ppml/examples/start-backend-sgx.sh
```

Comment:
Consider putting these scripts in separate folders

Reply:
Done.